app/models/manageiq/providers/vmware/infra_manager/inventory/parser.rb

```diff
@@ -114,7 +114,7 @@ def parse_distributed_virtual_switch(object, kind, props)
       dvpg_props.fetch_path(:config, :distributedVirtualSwitch)&._ref == object._ref
     end.each do |mor, dvpg_props|
       portgroup = RbVmomi::VIM::DistributedVirtualPortgroup(object._connection, mor)
-      parse_distributed_virtual_portgroup(portgroup, kind, dvpg_props)
+      parse_portgroups_internal(portgroup, dvpg_props)
     end
```

Comment:
This block was already a workaround for targeted update of a switch clearing the lans because of the same modeling issue